### PR TITLE
base16-schemes: unstable-2024-01-14 -> unstable-2024-06-21

### PR DESCRIPTION
**Pull request by IamPyu**
## Description of changes

Update base16-schemes package to latest commit [here](https://github.com/tinted-theming/schemes)

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [X] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/data/themes/base16-schemes/default.nix [hunk 1]
```diff
@@ -7,8 +7,8 @@ stdenv.mkDerivation (finalAttrs: {
   src = fetchFromGitHub {
     owner = "tinted-theming";
     repo = "schemes";
```
Comment (AndersonTorres) [inline pkgs/data/themes/base16-schemes/default.nix:9]: Update version.

**Review by AndersonTorres — CHANGES_REQUESTED** [review]
Reword the commit message according to contributing guidelines.

```
base16-schemes: <old  ver> -> <new ver>
```

File: pkgs/data/themes/base16-schemes/default.nix (hunk 1 above)
Comment (DamienCassou) [reply pkgs/data/themes/base16-schemes/default.nix:9]: Yes, we should replace "unstable-2024-01-14"; with the date corresponding to the commit.

File: pkgs/data/themes/base16-schemes/default.nix [hunk 2]
```diff
@@ -7,8 +7,8 @@ stdenv.mkDerivation (finalAttrs: {
...
-    rev = "395074124283df993571f2abb9c713f413b76e6e";
```
Comment (AndersonTorres) [inline pkgs/data/themes/base16-schemes/default.nix:10]: This is not the version, but the `version =` line above.

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/data/themes/base16-schemes/default.nix (hunk 2 above)
Comment (IamPyu) [reply pkgs/data/themes/base16-schemes/default.nix:10]: The package doesn't have versions, I just picked a commit from the master branch.

Comment (AndersonTorres) [reply pkgs/data/themes/base16-schemes/default.nix:10]: When the package has no version, we fill the `version` field with a string like `0-unstable-YYYY-MM-DD`, where the date is the same date when the commit you extracted was made.

Comment (IamPyu) [reply pkgs/data/themes/base16-schemes/default.nix:10]: OK.

**Review by DamienCassou — APPROVED** [review]

**Review by DamienCassou — CHANGES_REQUESTED** [review]
Please squash your commits into just one and make sure the commit conventions: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#commit-conventions.

**Review by AndersonTorres — CHANGES_REQUESTED** [review]
Squash all commits and reword as
```
base16-schemes: unstable-2024-01-14 -> unstable-2024-06-21
```

**Comment by AndersonTorres** [comment]
P.S.: the GitHub web interface is subpar.

**Review by AndersonTorres — COMMENTED** [review]
Not tested yet, but lgtm

**Comment by DamienCassou** [comment]
Result of `nixpkgs-review pr 321853` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>base16-schemes</li>
  </ul>
</details>

**Comment by DamienCassou** [comment]
Works for me.

Thank you for your work @IamPyu.

**Review by DamienCassou — APPROVED** [review]
LGTM

**Pull request merged**